e2e/cmd/keys.go

```diff
@@ -0,0 +1,252 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"reflect"
+
+	"github.com/omni-network/omni/e2e/app"
+	"github.com/omni-network/omni/e2e/app/eoa"
+	"github.com/omni-network/omni/e2e/app/key"
+	"github.com/omni-network/omni/e2e/types"
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/log"
+	"github.com/omni-network/omni/lib/netconf"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/naoina/toml"
+	"github.com/spf13/cobra"
+)
+
+func newKeyCreate(def *app.Definition) *cobra.Command {
+	cfg := key.UploadConfig{}
```

Comment:
```suggestion
	var cfg key.UploadConfig
```

Reply:
I just copied this to another file, didn't change the code, you are welcome to fix thanks